lib/store_attribute/active_record/store.rb

```diff
@@ -136,6 +137,8 @@ def store_attribute(store_name, name, type = :value, prefix: nil, suffix: nil, *
         _local_typed_stored_attributes[store_name][:types][name] = [type, options]
 
         if store_attribute_register_attributes
+          _ensure_attributes_sync_module_included
```

Comment:
Let's do this only if the feature is enabled (we probably can move the inclusion into the accessor method).